geos.c

```diff
@@ -19,6 +19,28 @@ void go_geos_error_handler(const char *fmt, ...) {
     va_end(ap);
 }
 
+void go_geos_LinearRingToFlatPoints (GEOSContextHandle_t handler, double out[], size_t n_out, const GEOSCoordSequence *seq) {
+    int i;
```

Comment:
Not a fan of having references to uninitialized memory regions, can you initialize this to `0` and then do nothing on the loop start?

Reply:
Fixed